pages/auth/verify-account.tsx

```diff
@@ -0,0 +1,116 @@
+import { AuthLayout } from '@/components/layouts';
+import { Alert, InputWithLabel } from '@/components/shared';
+import { defaultHeaders } from '@/lib/common';
+import { useFormik } from 'formik';
+import { GetServerSidePropsContext, InferGetServerSidePropsType } from 'next';
+import { serverSideTranslations } from 'next-i18next/serverSideTranslations';
+import { useRouter } from 'next/router';
+import { useState, type ReactElement, useEffect } from 'react';
+import { Button } from 'react-daisyui';
+import type { ComponentStatus } from 'react-daisyui/dist/types';
+import { toast } from 'react-hot-toast';
+import { useTranslation } from 'react-i18next';
+import { ApiResponse, NextPageWithLayout } from 'types';
+import * as Yup from 'yup';
+
+const VerifyAccount: NextPageWithLayout<
+  InferGetServerSidePropsType<typeof getServerSideProps>
+> = () => {
+  const router = useRouter();
+  const { t } = useTranslation('common');
+  const [message, setMessage] = useState<{
+    text: string | null;
+    status: ComponentStatus | null;
+  }>({
+    text: null,
+    status: null,
+  });
+
+  const { error } = router.query as { error: string };
+
+  useEffect(() => {
+    if (error) {
+      setMessage({ text: error, status: 'error' });
+    }
+  }, [router, router.query]);
+
+  const formik = useFormik({
+    initialValues: {
+      email: '',
+    },
+    validationSchema: Yup.object().shape({
+      email: Yup.string().required().email(),
+    }),
+    onSubmit: async (values) => {
+      const response = await fetch('/api/auth/verify-account', {
+        method: 'POST',
+        headers: defaultHeaders,
+        body: JSON.stringify(values),
+      });
+
+      const json = (await response.json()) as ApiResponse;
+
+      if (!response.ok) {
+        toast.error(json.error.message);
+        return;
+      }
+
+      formik.resetForm();
+      toast.success(t('verify-account-link-sent'));
+      router.push('/login');
```

Comment:
After sending the verification email, let's redirect the users to `/auth/verify-email`.